mergify_cli/tests/ci/test_cli.py

```diff
@@ -38,7 +38,7 @@
                 "CIRCLE_REPOSITORY_URL": "https://github.com/user/repo",
                 "CIRCLE_SHA1": "3af96aa24f1d32fcfbb7067793cacc6dc0c6b199",
                 "CIRCLE_JOB": "JOB",
-                "MERGIFY_TESTS_TARGET_BRANCH": "main",
+                "GITHUB_REF_NAME": "main",
```

Comment:
nitpic, I would have removed this case to keep it consistent (we don't have `GITHUB_REF_NAME` on Circle CI)